src/main/java/frc/robot/Controls.java

```diff
@@ -102,15 +107,22 @@ private void configureDrivebaseBindings() {
                   double dt = 0.02;
                   // Vx Vy and Omega are really accelerations and not velocities.
                   ChassisSpeeds acceleration = diff.div(dt);
-                  double translationAccelMagnitude =
-                      Math.hypot(acceleration.vxMetersPerSecond, acceleration.vyMetersPerSecond);
-                  ChassisSpeeds translationLimit =
-                      acceleration.times(Math.min(1, MAX_ACCELERATION / translationAccelMagnitude));
-                  ChassisSpeeds rotationLimit =
-                      translationLimit.times(
-                          Math.min(
-                              1,
-                              MAX_ROTATION_ACCELERATION / translationLimit.omegaRadiansPerSecond));
+                  // double translationAccelMagnitude =
+                  //     Math.hypot(acceleration.vxMetersPerSecond, acceleration.vyMetersPerSecond);
+                  // ChassisSpeeds translationLimit =
+                  //     acceleration.times(
+                  //         Math.min(1, Math.abs(MAX_ACCELERATION / translationAccelMagnitude)));
+                  // ChassisSpeeds rotationLimit =
+                  //     translationLimit.times(
+                  //         Math.min(
+                  //             1,
+                  //             Math.abs(
+                  //                 MAX_ROTATION_ACCELERATION
+                  //                     / translationLimit.omegaRadiansPerSecond)));
+                  // This *should* be rotationLimit, but the acceleration limiting causes the
+                  // commanded speed to fall into the deadband. The proper fix is to do the deadband
+                  // first, which relies on us doing the deadband ourselves, which is being
+                  // difficult.
```

Comment:
Generally, I've seen the deadband applied on the input value (the `driverController` inputs above) with 
[MathUtil.applyDeadband](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/MathUtil.html#applyDeadband(double,double)) to avoid issues like this.